src/libraries/Microsoft.PowerFx.Core/Lexer/Tokens/IdentToken.cs

```diff
@@ -26,16 +26,14 @@ public class IdentToken : Token
         /// </summary>
         public DName Name { get; }
 
-        internal const string StrInterpIdent = "Concatenate";
-
-        internal IdentToken(string val, Span span)
+        internal IdentToken(string val, Span span, bool isNonSourceIdentToken = false)
```

Comment:
>isNonSourceIdentToken [](http://example.com/codeflow?start=56&length=21)

Can you add a comment explaining what 'isNonSourceIdentToken' is? I could only find out by looking for its callers.